packages/api/tests/modules/etablissements/etablissements.spec.ts

```diff
@@ -1,14 +1,14 @@
-import request = require("supertest");
-import getUserToken from "../../../../__helpers__/getUserToken";
-import etablissementService from "../../../../../src/modules/etablissements/etablissements.service";
-import statsService from "../../../../../src/modules/stats/stats.service";
-import { siretToSiren } from "../../../../../src/shared/helpers/SirenHelper";
-import getAdminToken from "../../../../__helpers__/getAdminToken";
-import associationsService from "../../../../../src/modules/associations/associations.service";
-import { BadRequestError } from "../../../../../src/shared/errors/httpErrors";
-import OsirisRequestEntityFixture from "../../../providers/osiris/__fixtures__/entity";
-import dauphinService from "../../../../../src/modules/providers/dauphin/dauphin.service";
-import { osirisRequestRepository } from "../../../../../src/modules/providers/osiris/repositories";
+import request from "supertest";
```

Comment:
```suggestion
import request = require("supertest");
```

Chez moi si c'est pas ça mon IDE râle

Reply:
L'inverse chez moi :/

Reply:
En soit on fait que des import ES6 donc je vois pas pourquoi spécifiquement pour supertest ça devrait changer. Je penche pour un pb config sur ton IDE ? @VSerain

Reply:
cf #1017 c'est moi qui ai un IDE trop tâtillon

Reply:
Coucou, je vois pas trop ce sur quoi je peux vous apporter mon aide ?

Reply:
On a trouvé depuis, c'est bon ^^ Tant qu'on comprenait pas, un cerveau de + sur le coup paraissait utile

Reply:
Parfait !